mythril/laser/ethereum/strategy/__init__.py

```diff
@@ -1,12 +1,39 @@
 from abc import ABC, abstractmethod
 
+from mythril.laser.ethereum.graph import SimpleGraph, Graph
+
 
 class BasicSearchStrategy(ABC):
```

Comment:
Its getting to be a BaseSearchStrategy, vs BasicSearchStrategy. I think we should rename the class

Reply:
WDYT?

Reply:
But this isn't the Baseclass for all the search strategies right?but we can make it base class for AdvancedSearchStrategy WDYT?